db.go

```diff
@@ -776,19 +772,28 @@ func (db *DB) Query(q interface{}, args ...interface{}) (*Rows, error) {
 }
 
 // Rewrite the query to include a deadline if one is present in the context.
-func rewriteQuery(ctx context.Context, db *DB, start time.Time, q interface{}) (interface{}, error) {
+func rewriteQuery(ctx context.Context, db *DB, start time.Time, q string) (string, error) {
```

Comment:
Previously this would have taken an interface and the rewriter would have stringified the query. However, this would not work if you had `AllowStringQueries` set to false, since the second writer would run it though as a string and that would error.